conbench/api/results.py

```diff
@@ -157,96 +137,108 @@ def get(self) -> f.Response:
             name: name
             schema:
               type: string
+            description: Filter results to one specific conceptual benchmark name.
           - in: query
             name: batch_id
             schema:
               type: string
+            description: Filter results to one specific `batch_id`.
           - in: query
             name: run_id
             schema:
               type: string
+            description: |
+                Filter results to one specific `run_id`. Using this argument allows the
+                response to return results from before `2023-06-03 UTC`.
           - in: query
             name: run_reason
             schema:
               type: string
+            description: Filter results to one specific `run_reason`.
           - in: query
-            name: days
+            name: cursor
             schema:
-              type: integer
+              type: string
+            description: |
+                Cursor for pagination. To get the first page of results, leave out this
+                query parameter or submit `null`. The response's `metadata` key will
+                contain a `next_page_cursor` key, which will contain the cursor to
+                provide to this query parameter in order to get the next page. (If there
+                is expected to be no data in the next page, the `next_page_cursor` will
+                be `null`.)
+
+                The first page will contain the `page_size` most recent results matching
+                the given filter(s). Each subsequent page will have up to `page_size`
+                results, going backwards in time in DB insertion order, until there are
```

Comment:
I think this point is important enough it deserves to be in the top-level description; it's getting buried here